openmc/material.py

```diff
@@ -352,8 +353,8 @@ def add_macroscopic(self, macroscopic):
             self._macroscopic = macroscopic
         else:
             msg = 'Unable to add a Macroscopic to Material ID="{0}", ' \
-                  'Only One Macroscopic allowed per ' \
-                  'Material!'.format(self._id, macroscopic)
+                  'Only one Macroscopic allowed per ' \
+                  'Material!'.format(self._id)
```

Comment:
Can you remove the exclamation in this comment? Maybe it's just me, but I don't find the message any more exciting than our other error checking messages :-)